packages/qwik/src/core/render/execute-component.ts

```diff
@@ -119,12 +121,14 @@ export const pushRenderContext = (ctx: RenderContext): RenderContext => {
   return newCtx;
 };
 
-export const serializeClass = (
-  obj: string | { [className: string]: boolean } | (string | { [className: string]: boolean })[]
-): string => {
+export const serializeClass = (obj: ClassList): string => {
   if (!obj) return '';
   if (isString(obj)) return obj.trim();
 
+  if (qDev) {
+    verifySerializable(obj);
```

Comment:
Is it necessary?

Reply:
probably not! we are learning that this check is better done at the `jsx()` level, so when it crashes the stacktrace is actually helpful for the developer!